apps/docs/content/components/collections/TagGroup.mdx

```diff
@@ -1,49 +1,211 @@
 ---
 title: TagGroup
-description: TBD
+description: The TagGroup is a dynamic UI component that encapsulates a collection of tags.
```

Comment:
but I do think we should say what the tags represent or could be used for.. Any idea of how we should fit that in?

Reply:
My current issue is that it's too long in the overview page. With guidelines coming from Francis I expect that we will add more context in a coming section. Otherwise we could split description and teaser. Otherwise we have situations where 
![image](https://github.com/user-attachments/assets/8e606714-3158-463b-b5f7-8e9848da2799)
the preview text is long and not that useful when all you want is figure out is it's the component you are looking for.

Reply:
I can keep the description for the component page and add a concept of tagline that is shorter more to the point.